snap/info.go

```diff
@@ -421,8 +421,9 @@ func (s *Info) Services() []*AppInfo {
 	return svcs
 }
 
-// ExpandSnapVariables resolves $SNAP, $SNAP_DATA and $SNAP_COMMON.
-func (s *Info) ExpandSnapVariables(path string) string {
+// ExpandSnapMountVariables resolves $SNAP, $SNAP_DATA and $SNAP_COMMON inside the
+// snap's mount namespace.
+func (s *Info) ExpandSnapMountVariables(path string) string {
```

Comment:
The Mount doesn't add clarity here. The name here comes from "$SNAP_ variables". I know what you're going after, but there's just too much "mounting" happening for this to be useful. I suggest preserving the name for now.

Reply:
Ok, I'll go with updated comment then.